docs/conf.py

```diff
@@ -64,7 +66,7 @@
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
+language = "en-US"
```

Comment:
```suggestion
language = "en"
```

This should be `"en"` according to the docs below

- https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-language